src/drive/web/modules/drive/DeleteConfirm.jsx

```diff
@@ -61,4 +79,4 @@ const DeleteConfirmWithSharingContext = ({ files, ...rest }) =>
     </SharedDocument>
   )
 
-export default translate()(DeleteConfirmWithSharingContext)
+export default DeleteConfirmWithSharingContext
```

Comment:
Remark: theere is a codemod to do the change from translate() to useI18n() automatically. It might be good to run on the whole drive codebase.

```
jscodeshift --parser babel -t ~/code/cozy/cozy-ui/codemods/use-i18n-hook/index.js src --extensions js,jsx
```

Reply:
Right, i'll do a separate PR for that!